varcode/variant.py

```diff
@@ -681,3 +681,24 @@ def _exonic_transcript_effect(self, exon, exon_number, transcript):
                 exon=exon,
                 alternate_effect=coding_effect_annotation)
         return coding_effect_annotation
+
+    @property
+    def is_insertion(self):
+        """
+        Does this variant represent the insertion of nucleotides into the
+        reference genome?
+        """
+        return (len(self.ref) < len(self.alt)) and self.alt.startswith(self.ref)
```

Comment:
Yeah, it's done in the constructor but that's always felt a little controversial (since we're changing the representation away from how it's commonly displayed or what you see in VCFs) and I wanted to limit reliance on that normalization just in the variant effect prediction (where it matters the most). 

However, if you think that's more confusing then I can just simply the checks here and add comments saying we're assuming the variants have been normalized.